src/core/emulator.hpp

```diff
@@ -147,6 +149,8 @@ class Emulator
 
         void test_iop();
         GraphicsSynthesizer& get_gs();//used for gs dumps
+        EEBreakpointList* get_ee_breakpoint_list();
+        IOPBreakpointList* get_iop_breakpoint_list();
```

Comment:
Recommend marking as `const`.

```suggestion
        IOPBreakpointList* get_iop_breakpoint_list() const;
```